code/modules/mob/living/carbon/human/human.dm

```diff
@@ -380,16 +380,24 @@
 					return
 
 	//SKYRAT EDIT ADDITION BEGIN - VIEW RECORDS
-	if(href_list["bgrecords"])
+	var/examined_name = get_face_name(get_id_name(""))
+	var/datum/record/locked/target_locked_record = find_record(examined_name, TRUE)
+	var/datum/record/crew/target_record = find_record(examined_name, FALSE)
+	if(href_list["secrecords"] && target_record)
+		if(isobserver(usr))
+			to_chat(usr, "<b>Security Record:</b> [target_record.past_security_records]")
+	if(href_list["medrecords"] && target_record)
+		if(isobserver(usr))
+			to_chat(usr, "<b>Medical Record:</b> [target_record.past_medical_records]")
+	if(href_list["genrecords"] && target_record)
+		if(isobserver(usr))
+			to_chat(usr, "<b>General Record:</b> [target_record.past_general_records]")
+	if(href_list["bgrecords"] && target_locked_record)
+		if(isobserver(usr))
+			to_chat(usr, "<b>Background information:</b> [target_locked_record.background_information]")
+	if(href_list["exprecords"] && target_locked_record)
 		if(isobserver(usr) || usr.mind.can_see_exploitables || usr.mind.has_exploitables_override)
```

Comment:
can't really be done more efficient for only this one but should be reordered either way
```suggestion
	if(isobserver(usr) || usr.mind.can_see_exploitables || usr.mind.has_exploitables_override)
		if(target_locked_record && href_list["exprecords"])
```